### ISSUE TITLE
[Feature]:

### ISSUE THREAD

**Issue by denis-v-s** [NuvioMedia/NuvioDesktop#24]
### Area (tag)

UI / Layout

### Problem statement

Lack of keyboard navigation

### Proposed solution

Add support for movie/show navigation with keyboard shortcuts / arrow keys.
Highlight active movie/show

### Are you planning to implement this yourself?

No, this is only a proposal
